traffic_monitor_golang/traffic_monitor/health/cache.go

```diff
@@ -201,7 +201,7 @@ func CalcAvailability(results []cache.Result, pollerName string, statResultHisto
 
 		localStates.SetCache(result.ID, peer.IsAvailable{IsAvailable: isAvailable})
 	}
-	calculateDeliveryServiceState(toData.DeliveryServiceServers, localStates)
+	// calculateDeliveryServiceState(toData.DeliveryServiceServers, localStates)
```

Comment:
It looks like the primary job of this func was to compute DisabledLocations. Is `disabledLocations` in `CrStates` still correctly populated?